File: Python/Easy/are_true.py
# https://edabit.com/challenge/3ex9eCDEBqdxvfQvD

def are_true(a, b):
	return "both" if a == True and b == True else 'first' if a == True and b == False else "second" if a == False and b == True else 'neither'

# original code

# def are_true(a, b):
# 	if a == True:
# 		if b == True:
# 			return "both"
# 		else:
# 			return "first"
# 	elif b = True:
# 		return "second"
# 	else:
# 		return "neither"


print(are_true(True, True), "both")
print(are_true(True, False), "first")
print(are_true(False, True), "second")
print(are_true(False, False),'neither')

File: javascript/Very_Easy/areTrue.js
// https://edabit.com/challenge/j3aeuun5WDXFsTYAK


function areTrue(a, b) {
	if (a == true) {
		if (b == true) {
			return "both";
		}
		else {
			return "first";
		}
	}
	else if (b = true) {
		return "second";
	}
	else {
		return "neither";
	}
}

// function above adjusted to this using the ternatory operator

const areTrue =(a,b) => a && b ? 'both' : a && !b ? 'first': !a && b ? 'second': 'neither';

console.log(areTrue(true, true), "both")
console.log(areTrue(true, false), "first")
console.log(areTrue(false, true), "second")
console.log(areTrue(false, false), "neither")